docs/ja/features/ngsi-subscriptions.md

```
@@ -1,11 +1,684 @@
+## 目次
+
+
+* [概要](#概要)
+  
+* [サブスクリプションの仕組み](#サブスクリプションの仕組み)
+  
+* [通知方法](#通知方法)
+  
+* [条件とフィルタリング](#条件とフィルタリング)
+  
+* [実用的な例](#practical-examples)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)